tests/test_save_load.py

```diff
@@ -301,3 +335,77 @@ def test_save_load_policy(tmp_path, model_class, policy_str):
     os.remove(tmp_path / "policy.pkl")
     if actor_class is not None:
         os.remove(tmp_path / "actor.pkl")
+
+
+@pytest.mark.parametrize("pathtype", [str, pathlib.Path])
```

Comment:
Also test providing BufferedIOBase object, e.g. like [done in stable-baselines](https://github.com/hill-a/stable-baselines/blob/master/tests/test_save.py#L82). I understood this should now also work with this code.